owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/settings/SettingsSecurityViewModelTest.kt

```diff
@@ -139,28 +140,18 @@ class SettingsSecurityViewModelTest : ViewModelTest() {
 
     @Test
     fun `set pref is security enforced enabled - ok - true`() {
-        every { contextProvider.getBoolean(any()) } returns true
+        every { contextProvider.getInt(any()) } returns LockEnforcedType.EITHER_ENFORCED.ordinal
 
-        securityViewModel.isSecurityEnforcedEnabled().apply {
-            assertTrue(this)
-        }
+        val result = securityViewModel.isSecurityEnforcedEnabled()
+        assertFalse(result)
```

Comment:
If we have a `LockEnforcedType.EITHER_ENFORCED`, how can `securityViewModel.isSecurityEnforcedEnabled()` be `false`?

Am I missing something?

Reply:
Fixed, thanks!